lib/beaker/options/parser.rb

```diff
@@ -165,6 +166,34 @@ def initialize
         @command_line_parser = Beaker::Options::CommandLineParser.new
         @presets             = Beaker::Options::Presets.new
         @validator           = Beaker::Options::Validator.new
+        @attribution         = Beaker::Options::OptionsHash.new
```

Comment:
@kevpl sorry man, this update was mainly aimed at recursively updating the attribution hash for child items, e.g. the SSH attribute. I intended to make further updates to encapsulate the attribution hash within the OptionsHash, but I got sidetracked with some CI.next stuff. I'll work on these changes today